it/uk/gov/hmrc/transitmovementsconverter/itbase/TestActorSystem.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsconverter.itbase
+
+import akka.actor.ActorSystem
+import akka.stream.Materializer
+import org.scalatest.Suite
```

Comment:
Perhaps consider:
"com.typesafe.akka" %% "akka-stream-testkit" % akkaVersion

Reply:
We'd still need to create an actor system, the test kit is for probing stuff. It might be useful elsewhere, I don't think it's particularly useful here (particularly in this test where we just want the output, we don't need to probe anything other than that).